metasync.js

```diff
@@ -4,23 +4,23 @@ const common = require('@metarhia/common');
 const nodeVerion = common.between(process.version, 'v', '.');
 
 const submodules = [
-  'composition', // Unified abstraction
-  'control', // Control flow utilities
-  'fp', // Async utils for functional programming
   'adapters', // Adapters to convert different async contracts
-  'throttle', // Throttling utilities
   'array', // Array utilities
   'chain', // Process arrays sync and async array in chain
   'collector', // DataCollector and KeyCollector
-  'queue', // Concurrent queue
-  'memoize', // Async memoization
+  'composition', // Unified abstraction
+  'control', // Control flow utilities
   'do', // Simple chain/do
+  'fp', // Async utils for functional programming
+  'memoize', // Async memoization
   'poolify', // Create pool from factory
+  'queue', // Concurrent queue
+  'throttle', // Throttling utilities
 ].map(path => require('./lib/' + path));
 
 if (nodeVerion >= 10) {
   submodules.push(require('./lib/async-iterator'));
 }
 
-const { compose } = submodules[0];
+const { compose } = submodules[4];
```

Comment:
I think we can leave the `'composition'` the first element of the array to avoid the need to count the elements of the array to understand from which module `compose` is destructured.